### PR TITLE
Set controller internal timeouts at startup

### PR DESCRIPTION
**Pull request by parrotmac**

### CODE REVIEW

File: serialapi/controller_timeouts.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	"github.com/gozwave/gozw/session"
+)
+
+type ControllerTimeouts struct {
+	AckTimeout  byte
+	ByteTimeout byte
+}
+
+const (
+	ackTimeout  = 0x96 // 0x95 = 150 -- multiply by 10 to get 1500ms
```
Comment (zacatac) [inline serialapi/controller_timeouts.go:17]: Could you put a comment in here with a link to the documentation on this? (Even if it's just the section header)

File: serialapi/controller_timeouts.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	byteTimeout = 0x0F // 0x0F = 15 -- multiply by 10 to get 150ms
+
+	errUnableToSetControllerTimeouts = "unable to set controller timeouts"
+)
+
+func (s *Layer) SetControllerTimeouts() (*ControllerTimeouts, error) {
+	done := make(chan *frame.Frame)
+
+	request := &session.Request{
+		FunctionID: protocol.FnSerialAPIGetTimeouts,
```
Comment (zacatac) [inline serialapi/controller_timeouts.go]: Don't you want this to be `FnSerialAPISetTimeouts`?

**Review by zacatac — APPROVED** [review]
Couple of questions, but generally makes sense.

File: serialapi/controller_timeouts.go (hunk 2 above)
Comment (parrotmac) [reply serialapi/controller_timeouts.go]: Yeah, not sure how it ended up there as 'Get'

**Pull request merged**